.github/workflows/develop_commit_notification.yml

```diff
@@ -6,10 +6,10 @@ on:
 
 jobs:
   send_notifications:
-    name: Repository Dispatch
-    uses: peter-evans/repository-dispatch@v2
-    with:
-      token: ${{ secrets.REPO_ACCESS_TOKEN }}
-      repository: oppia/release-scripts
-      event-type: develop-commit
-      client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}"}'
+    - name: Repository Dispatch
```

Comment:
can we restrict this to the oppia/oppia repo alone?

Reply:
Not really we want the build process to be kind of hidden in the release-scripts repository.

Reply:
can we try something like this? https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution

so this action would only run on the Oppia repo and not on forks?

Reply:
Oh, right, makes sense, thanks for the link.

Reply:
Done.